api/src/test/java/org/pmiops/workbench/user/UserAdminServiceTest.java

```diff
@@ -103,4 +104,40 @@ public void testGetActiveWindow_noActiveWindow_null() {
     // does not throw
     assertThat(userAdminService.getCurrentEgressBypassWindow(null)).isNull();
   }
+
+  @Test
+  public void tesListWindows() {
+    Instant startTime1 = FakeClockConfiguration.NOW.toInstant().minus(1, ChronoUnit.DAYS);
+    Instant endTime = FakeClockConfiguration.NOW.toInstant().plus(1, ChronoUnit.DAYS);
+    Instant startTime2 = startTime1.plus(1, ChronoUnit.DAYS);
+    DbUserEgressBypassWindow dbUserEgressBypassWindow1 =
+        new DbUserEgressBypassWindow()
+            .setUserId(USER_ID)
+            .setStartTime(Timestamp.from(startTime1))
+            .setEndTime(Timestamp.from(endTime))
+            .setDescription(DESCRIPTION);
+    DbUserEgressBypassWindow dbUserEgressBypassWindow2 =
+        new DbUserEgressBypassWindow()
+            .setUserId(USER_ID)
+            .setStartTime(Timestamp.from(startTime2))
+            .setEndTime(Timestamp.from(endTime))
+            .setDescription(DESCRIPTION);
+    userEgressBypassWindowDao.saveAll(
+        ImmutableList.of(dbUserEgressBypassWindow1, dbUserEgressBypassWindow2));
+    assertThat(userAdminService.listAllEgressBypassWindows(USER_ID))
+        .containsExactly(
```

Comment:
Please add .`inOrder()` after this call, to show the ordering of the results.  I expect that this will fail as written, because the later window should appear first.

Reply:
Done